wasp/widgets.py

```diff
@@ -285,3 +285,80 @@ def touch(self, event):
         elif v >= self._steps:
             v = self._steps - 1
         self.value = v
+
+
+_message_string_x_coord = const(0)
+_message_string_y_coord = const(60)
+_yes_button_x_coord = const(20)
+_yes_button_y_coord = const(100)
+_no_button_x_coord = const(120)
+_no_button_y_coord = const(100)
+
+class ConfirmationView:
+    "Confirmation widget allowing user confirmation of a setting"
+
+    class TouchResult:
+        YES = 0
+        NO = 1
+        NO_ACTION = 2
```

Comment:
Do we really need this special type? It looks like these are synonyms for True, False, None .

Reply:
Have changed to your suggestion, I felt that it was better to be explicit over the what the three states represent rather then just returning bools/nulls. The other advantage is there's a distinct value difference then between the types, where False and None can be lumped togeather as falsy